packages/jest/src/create-serializer.js

```diff
@@ -89,15 +89,17 @@ function filterEmotionProps(props = {}) {
   return rest
 }
 
+function getStylesFromCss(css) {
+  return (Array.isArray(css) ? css : [css]).filter(Boolean)
```

Comment:
`css` can contain nested arrays etc so I think that you should let things through and handle them more granularly - when there is an attempt to operate on an item. That would allow for a natural flow of the program - rather than purifying this recursively upfront.

Reply:
I updated my branch per your suggestion.